policyengine-client/src/policyengine/pages/policy/policy.jsx

```diff
@@ -54,15 +54,15 @@ export default class Policy extends React.Component {
 				        </Affix>
                         <Affix offsetTop={450}>
                             <SharePolicyLinks page="policy"/>
-                            <div className="d-block align-middle">
-                                <div className="d-flex justify-content-center">
+                            <div className="d-block align-middle mx-5">
```

Comment:
I think we can remove the `mx` class here - it seems to look the same without, and it for some reason removes the entire sidebar on the household page.